### ISSUE TITLE
"torch.nn.functional.interpolate" gives corrupted output

### ISSUE THREAD

**Issue by m-pektas** [NVIDIA-AI-IOT/torch2trt#525]
## Description

I have a custom autoencoder network. I use them as two different part. When I convert my encoder part from torch to tensorrt fp16, It works correctly. But, When I did same process for decoder part, It is not working. (Not working correctly means, it mostly give correct result but sometimes output is corrupted at inference time). What could be reason of this problem ? Furthermore, They contains almost same component. One of the diffirent component between them is **"torch.nn.functional.interpolate"**.  I tried to convert only this function with torch2trt, it gives me corrupted result. Of course, original function (not converted) works correctly. Also I tried same converting operations with sigmoid, It worked on both methods (with torch2trt/ only torch). The converted module and results are shown following images.

## Module

![Screenshot from 2021-03-12 20-09-16](https://user-images.githubusercontent.com/15729405/110973860-dd74e980-836e-11eb-9d98-1a775fb31a11.png)

## Results

![interpolate_error_torch2trt](https://user-images.githubusercontent.com/15729405/110973295-1f516000-836e-11eb-829b-a9a9f16499b9.jpeg)

**Comment by poornimajd** [comment]
@m-pektas were you able to find a workaround for this?

**Comment by sapjunior** [comment]
I also encounter this problem as well (fp32 is acceptable but not for fp16 and int8) I seem like the source of problem come from TensorRT itself as mentioned in #https://github.com/NVIDIA/TensorRT/issues/1029#issuecomment-768122228

**Comment by m-pektas** [comment]
Interpolation problem was caused by wrong input resolution. But still, Some outputs are corrupted in batch when we used fp16 tensorRT model at inference time.
